ops/terraform/services/common/s3-test-data.tf

```diff
@@ -0,0 +1,108 @@
+
+data "aws_canonical_user_id" "current" {}
+
+# bfd-public-test-data S3 bucket imports, not currently managed with terraform
+import {
+  to = aws_s3_bucket.bfd-public-test-data
+  id = "bfd-public-test-data"
+}
+
+import {
+  to = aws_s3_bucket_server_side_encryption_configuration.bfd-public-test-data
+  id = "bfd-public-test-data"
+}
+
+import {
+  to = aws_s3_bucket_versioning.bfd-public-test-data
+  id = "bfd-public-test-data"
+}
+
+resource "aws_s3_bucket" "bfd-public-test-data" {
+  bucket = "bfd-public-test-data"
+
+  tags = {
+    "cms-cloud-exempt:public-s3-bucket" = "https://confluence.cms.gov/x/VFNBGg"
+  }
+}
+
+resource "aws_s3_bucket_server_side_encryption_configuration" "bfd-public-test-data" {
+  bucket = aws_s3_bucket.bfd-public-test-data.id
+  rule {
+    apply_server_side_encryption_by_default {
+      sse_algorithm = "AES256"
+    }
+  }
+}
+
+resource "aws_s3_bucket_versioning" "bfd-public-test-data" {
+  bucket = aws_s3_bucket.bfd-public-test-data.id
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
+
+resource "aws_s3_bucket_lifecycle_configuration" "bfd-public-test-data" {
+  depends_on = [aws_s3_bucket_versioning.bfd-public-test-data]
+  bucket = aws_s3_bucket.bfd-public-test-data.id
+
+  rule {
+    id = "rule-1"
+
+    filter {}
+
+    noncurrent_version_transition {
+      noncurrent_days = 60
+      storage_class   = "GLACIER"
```

Comment:
Maybe a better option here would be [s3 Intelligent Tiering](https://aws.amazon.com/s3/storage-classes/). This would automatically move objects to cheaper storage depending on usage. Ie, objects not used in 30 days get moved to a cheaper tier, same for 60, and cold storage after 90. Cost is similar to Glacier without the retrieval penalty. @mjburling - I was researching this earlier as it may be a good fit for most if not all of our buckets (eft, etl, etc).

Reply:
$.10 per million objects fee for tracking object usage (https://aws.amazon.com/s3/pricing/?nc=sn&loc=4)